var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Migraphx(CMakePackage):
+    """ AMD's graph optimization engine."""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX"
+
```

Comment:
I would still include a URL up here, it's used by `spack versions` and `spack checksum` to locate new versions.

Reply:
Just wanted to know if you meant the below
 homepage = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX"
    url = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/rocm-3.5.0.tar.gz"

    maintainers = ['srekolam', 'arjun-raj-kuppala']

    def url_for_version(self, version):
        url = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/{0}.tar.gz"
        if version > Version('3.5'):
            return url.format(version)
        else:
            return url.format('rocm-' + version)

Reply:
Yes, just put it below the homepage